Hi!

http://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=250492
requests adding transliteration of Polish [lL] with stroke,
which makes sense to me, e.g. Slovak [lL] with caron is
already transliterated as [lL].

 2007-08-03  Jakub Jelinek  <jakub@redhat.com>

	* locales/translit_combining: Add U0141 and U0142.

===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/translit_combining,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/localedata/locales/translit_combining	2002/04/20 07:12:59	1.3
+++ libc/localedata/locales/translit_combining	2007/08/03 15:41:17	1.4
@@ -472,6 +472,10 @@
 <U013D> <U004C>
 % LATIN SMALL LETTER L WITH CARON
 <U013E> <U006C>
+% LATIN CAPITAL LETTER L WITH STROKE
+<U0141> <U004C>
+% LATIN SMALL LETTER L WITH STROKE
+<U0142> <U006C>
 % LATIN CAPITAL LETTER N WITH ACUTE
 <U0143> <U004E>
 % LATIN SMALL LETTER N WITH ACUTE
